### PR TITLE
Added Manifest file for AWS SDK Bundle

### PR DESCRIPTION
**Pull request by trandangtri**
| Q             | A
| ------------- | ---
| License       | MIT

Manifest for AWS SDK. This one closes for an issue: https://github.com/symfony/flex/issues/193

### CODE REVIEW

File: aws/aws-sdk-php-symfony/1.3/config/packages/aws.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Basic configuration for AWS SDK
+# Please visit http://docs.aws.amazon.com/aws-sdk-php/v3/guide/guide/configuration.html
+# for more information
+
+aws:
+    version: latest
+    region: us-east-1
+    credentials:
+        key: not-a-real-key
+        secret: "@@not-a-real-secret"
```
Comment (fabpot) [inline aws/aws-sdk-php-symfony/1.3/config/packages/aws.yaml]: `key` and `secret` should probably be env vars instead.

File: aws/aws-sdk-php-symfony/1.3/config/packages/aws.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Basic configuration for AWS SDK
+# Please visit http://docs.aws.amazon.com/aws-sdk-php/v3/guide/guide/configuration.html
+# for more information
```
Comment (fabpot) [inline aws/aws-sdk-php-symfony/1.3/config/packages/aws.yaml]: For the comment, can you get some inspiration from existing comments in for other packages (just that they are all consistent?).

Comment (trandangtri) [reply aws/aws-sdk-php-symfony/1.3/config/packages/aws.yaml]: Thanks you. I think I should simply remove it, like the other packages for consistent.

File: aws/aws-sdk-php-symfony/1.3/config/packages/aws.yaml (hunk 1 above)
Comment (trandangtri) [reply aws/aws-sdk-php-symfony/1.3/config/packages/aws.yaml]: Really helpful. Thanks

**Comment by trandangtri** [comment]
Thanks @fabpot for your comments. I fixed all of them like your suggestions, but still waiting for approval from the gents at AWS. We could come back to this PR later.

File: aws/aws-sdk-php-symfony/1.3/manifest.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+    "bundles": {
+        "Aws\\Symfony\\AwsBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "AWS_CREDENTIAL_KEY": "not-a-real-key",
```
Comment (fabpot) [inline aws/aws-sdk-php-symfony/1.3/manifest.json]: Could we use just `AWS_KEY` and `AWS_SECRET` or are those names recommended by AWS? Shorter is better IMHO.

Comment (trandangtri) [reply aws/aws-sdk-php-symfony/1.3/manifest.json]: Done :)

**Comment by fabpot** [comment]
Blocked by aws/aws-sdk-php-symfony#30

**Comment by trandangtri** [comment]
Hi @fabpot and team. Please re-execute the checking, thanks so much

**Comment by fabpot** [comment]
@symfony-flex-server review please

**Comment by fabpot** [comment]
@symfony-flex-server review please

**Pull request merged**